core/src/main/java/com/adyen/checkout/core/redirect/internal/ui/RedirectEvent.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2025 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by ararat on 7/8/2025.
+ */
+
+package com.adyen.checkout.core.redirect.internal.ui
+
+import androidx.annotation.RestrictTo
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.ui.platform.LocalContext
+import com.adyen.checkout.core.common.AdyenLogLevel
+import com.adyen.checkout.core.common.internal.helper.adyenLog
+import com.adyen.checkout.core.redirect.internal.RedirectHandler
+import kotlinx.coroutines.flow.Flow
+
+@Suppress("TooGenericExceptionCaught")
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+@Composable
+fun RedirectEvent(
```

Comment:
Because it's not a composable that renders UI, shouldn't it be starting with lowercase?